spec/src/main/asciidoc/changelog.adoc

```diff
@@ -19,75 +19,158 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-== Major changes to previous versions
+= Release Notes
+
 
 Changes marked with icon:bolt[role="red"] are breaking changes relative to previous versions of the spec.
 
-* Changes in 2.3
-** Introduced a new base metric derived from RESTful stats into the base scope.
```

Comment:
I can't find this in the new release notes, seems REST metrics got lost in the transition :)

Reply:
fixed :(